poreCov.nf

```diff
@@ -63,10 +63,10 @@ println "_______________________________________________________________________
 
 // Log infos based on user inputs
 if ( params.help ) { exit 0, helpMSG() }
-    defaultMSG()
-if ( params.primerV.matches('V1200') ) { v1200_MSG() }
-if ( params.fast5 || workflow.profile.contains('test_fast5') ) { basecalling() }
-if ( params.rki ) { rki() }
+//    defaultMSG()
```

Comment:
@DataSpott why is this commented out?

Reply:
Forgot to deletes this after moving to new section